### ISSUE TITLE
virtual consoles are English keyboard, (Bugzilla Bug 618)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#618]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 618
Date: 2014-03-25 17:25:33 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;
To:   @tpgxyz
CC:   agoiza@gmail.com, @cris-b, gio_6b@yahoo.it, @itchka, nelvirv@gmail.com, @tpgxyz

Depends on:   #669
Last updated: 2014-04-28 07:27:30 +0000

## Comment 4472
Date: 2014-03-25 17:25:33 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

beta x86

tty 1,2 3 etc have english keyboard

it's hard for strangers to log into console :)

## Comment 4474
Date: 2014-03-25 17:37:41 +0000
From: @tpgxyz

Run keyboarddrake, choose your language etc. and reboot

## Comment 4475
Date: 2014-03-25 17:48:34 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #1)
> Run keyboarddrake, choose your language etc. and reboot

hem, in X I am connected in french language and all work, it is not normal to have tty  in english

it was not like that in 2013 alpha ..

## Comment 4478
Date: 2014-03-25 18:33:51 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

As you can see the language is already the French, which is normal and corresponds to the installation.
Now you see what you do with what I consider a bug

## Comment 4479
Date: 2014-03-25 18:34:49 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 326
screenshot

keyboarddrake

> Attached file: capture d'écran1.png (image/png, 43633 bytes)
> Description:   screenshot

## Comment 4501
Date: 2014-03-26 09:21:07 +0000
From: @tpgxyz

What says /etc/vconsole.conf ?

## Comment 4504
Date: 2014-03-26 09:53:20 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #5)
> What says /etc/vconsole.conf ?

I can not show you the file because I am currently attempting an update of omv 2013 to 2014 beta

but I remember seeing KEYMAP = latin-fr1 something like that

## Comment 4506
Date: 2014-03-26 09:53:33 +0000
From: @tpgxyz

*** Bug #615 has been marked as a duplicate of this bug. ***

## Comment 4640
Date: 2014-03-28 21:44:37 +0000
From: @tpgxyz

(In reply to comment #6)
> (In reply to comment #5)
> > What says /etc/vconsole.conf ?
> 
> I can not show you the file because I am currently attempting an update of
> omv 2013 to 2014 beta
> 
> but I remember seeing KEYMAP = latin-fr1 something like that

This is related to 2013.0 ?

## Comment 4656
Date: 2014-03-29 05:24:46 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #8)
> (In reply to comment #6)
> > (In reply to comment #5)
> > > What says /etc/vconsole.conf ?
> > 
> > I can not show you the file because I am currently attempting an update of
> > omv 2013 to 2014 beta
> > 
> > but I remember seeing KEYMAP = latin-fr1 something like that
> 
> This is related to 2013.0 ?

no 2014 beta

# cat /etc/vconsole.conf
KEYMAP=fr-latin1

in tty (1,2,3 ..etc) before logging I have English keyboard, after logging in tty it is French keyboard

## Comment 4658
Date: 2014-03-29 07:26:16 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 351
omv-bug-report.log

2014.0 beta i586

Same result as Joel on i586.

Virtual console has default keyboard, depsite my keymap is italian.

$ cat /etc/vconsole.conf
KEYMAP=it-latin1

but remains "default" also after login in tty.

> Attached file: omv-bug-report.log (text/x-log, 267260 bytes)
> Description:   omv-bug-report.log

## Comment 4767
Date: 2014-04-02 15:29:52 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 361
omv-bug for 2014 RC build 3101

the issue still occurs in 2014 RC build 3101

> Attached file: omv-bug-report.log.xz (application/x-xz, 57692 bytes)
> Description:   omv-bug for 2014 RC build 3101

## Comment 4802
Date: 2014-04-03 07:27:31 +0000
From: @tpgxyz

According to your omv-bug-report a systemd-vconsole-setup.service is not started.

Please run as a root:

systemctl is-enabled systemd-vconsole-setup.service

if this is disabled then run

systemctl enable systemd-vconsole-setup.service

## Comment 4804
Date: 2014-04-03 07:32:14 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #12)
> According to your omv-bug-report a systemd-vconsole-setup.service is not
> started.
> 
> Please run as a root:
> 
> systemctl is-enabled systemd-vconsole-setup.service
> 
> if this is disabled then run
> 
> systemctl enable systemd-vconsole-setup.service

# systemctl is-enabled systemd-vconsole-setup.service
static
[root@localhost moi]# systemctl enable systemd-vconsole-setup.service
The unit files have no [Install] section. They are not meant to be enabled
using systemctl.
Possible reasons for having this kind of units are:
1) A unit may be statically enabled by being symlinked from another unit's
   .wants/ or .requires/ directory.
2) A unit's purpose may be to act as a helper for some other unit which has
   a requirement dependency on it.
3) A unit may be started when needed via activation (socket, path, timer,
   D-Bus, udev, scripted systemctl call, ...).

## Comment 4817
Date: 2014-04-03 11:19:10 +0000
From: @tpgxyz

Anyways looks like your installation is heavily broken, because this service is not started at all.

Please run binary manually by typing in you console this command:

/lib/systemd/systemd-vconsole-setup

## Comment 4828
Date: 2014-04-03 13:51:48 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #14)
> Anyways looks like your installation is heavily broken, because this service
> is not started at all.
> 
> Please run binary manually by typing in you console this command:
> 
> /lib/systemd/systemd-vconsole-setup

no, my installation is not strongly broken!
  It 'is a recent installation, no change on my part. 2 We refer to this error, you'd better ask you please "/ lib / systemd / systemd-VConsole-setup" is not started at boot somewhere.
Anyway, I do not continue to report errors, since it is useless

## Comment 4848
Date: 2014-04-03 23:00:27 +0000
From: @cris-b

confirming this on physical (2013->2014) and VM (fresh 2014).

looks like it could be something to do with KMS overriding the vconsole?

https://bbs.archlinux.org/viewtopic.php?id=146743
 
doing:

systemctl restart systemd-vconsole-setup.service

sets up the console ok.

## Comment 4865
Date: 2014-04-04 06:44:10 +0000
From: @tpgxyz

(In reply to comment #16)
> confirming this on physical (2013->2014) and VM (fresh 2014).
> 
> looks like it could be something to do with KMS overriding the vconsole?
> 
> https://bbs.archlinux.org/viewtopic.php?id=146743
>  
> doing:
> 
> systemctl restart systemd-vconsole-setup.service
> 
> sets up the console ok.

Have you tried to add to systemd-vconsole-setup.service these two lines:

After=systemd-udev-settle.service
Wants=systemd-udev-settle.service

and rebuild initrd ?

## Comment 4946
Date: 2014-04-06 09:03:02 +0000
From: @tpgxyz

Fixed in this commit 
https://abf.io/openmandriva/systemd/commit/a7fc6d55cfa8c1360b76d769f2783379d9d0ac6b

Please update to systemd-208-19 and rebuild your initrd by running dracut -f

## Comment 4952
Date: 2014-04-06 20:58:45 +0000
From: @cris-b

seems no better for me.

i've still got US keyboard until i manually execute systemd-vconsole-setup.

## Comment 4955
Date: 2014-04-07 06:38:46 +0000
From: @tpgxyz

(In reply to comment #19)
> seems no better for me.
> 
> i've still got US keyboard until i manually execute systemd-vconsole-setup.

Strange, can you provide output from systemd-analyze ?

## Comment 4956
Date: 2014-04-07 06:50:00 +0000
From: @tpgxyz

(In reply to comment #20)
> (In reply to comment #19)
> > seems no better for me.
> > 
> > i've still got US keyboard until i manually execute systemd-vconsole-setup.
> 
> Strange, can you provide output from systemd-analyze ?

Disregard my above question. Can you please boot in systemd debug mode (in grub submenu boot latest kernel with recovery mode) and attach outou of omv-bug-report.sh and output after systemctl start systemd-vconsole-setup.

I've found this bug, sounds related:
https://bugzilla.redhat.com/show_bug.cgi?id=1001411

## Comment 5000
Date: 2014-04-08 08:23:01 +0000
From: @tpgxyz

Raising importance

## Comment 5001
Date: 2014-04-08 08:57:34 +0000
From: @tpgxyz

(In reply to comment #22)
> Raising importance

According to some bug reports this starting sound like a kernel bug, see below:
https://bugzilla.redhat.com/show_bug.cgi?id=927564#c89
https://bugzilla.redhat.com/show_bug.cgi?id=970030#c14

There is still left this one to test:
https://bugzilla.redhat.com/show_bug.cgi?id=1001411#c23

## Comment 5019
Date: 2014-04-09 07:47:55 +0000
From: @tpgxyz

I've released dracut-037 with few fixes and settings like:

i18n_vars="/etc/locale.conf:SYSFONT-FONT,SYSFONTACM-FONT_MAP,UNIMAP-FONT_UNIMAP,LANG,LC_ALL-LANG /etc/vconsole.conf:KEYMAP,KEYTABLE,GRP_TOGGLE"
i18n_default_font="latarcyrheb-sun16"
i18n_install_all="yes"

Please rebuild your initrd.

Warining:
This release fixes bug https://issues.openmandriva.org/show_bug.cgi?id=651
which means that running "dracut -f" rebuild existing /boot/initrd-*.img in stead of /boot/initramfs-*.img

## Comment 5023
Date: 2014-04-09 08:40:19 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i have tried with dracut-037-1

=========================================================================
$ ls -alt /boot
totale 149304
drwxr-xr-x 22 root root     4096 apr  9 10:30 ../
-rw-------  1 root root 23010680 apr  9 10:28 initrd-3.13.8-nrjQL-desktop-1omv.img
....
....
=========================================================================

Still US keyboard until i manually execute systemd-vconsole-setup.

I've noticed:

dracut-install: ERROR: installing '/usr/bin/keyctl'

during /usr/sbin/dracut -f

# urpmi dracut-037-1-omv2014.0.i586.rpm 
I pacchetti tar-1.27-6-omv2014.0.i586, gzip-1.6-6-omv2014.0.i586, bzip2-1.0.6-14-omv2014.0.i586, e2fsprogs-1.42.8-2-omv2014.0.i586, coreutils-8.21-12-omv2014.0.i586, util-linux-2.23.2-8-omv2014.0.i586, systemd-208-19-omv2014.0.i586, binutils-2.23.52.0.2-8-omv2014.0.i586, setup-2.7.22-12-omv2014.0.i586, kbd-2.0.0-7-omv2014.0.i586, rpm-helper-0.24.12-8-omv2014.0.noarch, glibc-2.19-2-omv2014.0.i586, kmod-compat-16-2-omv2014.0.i586, dash-0.5.7-9-omv2014.0.i586, cpio-2.11-13-omv2014.0.i586, bash-4.2-24-omv2014.0.i586, findutils-4.5.11-8-omv2014.0.i586, file-5.15-7-omv2014.0.i586, initscripts-9.52-4-omv2014.0.i586, udev-208-19-omv2014.0.i586, v86d-0.1.10-3-omv2014.0.i586, update-alternatives-1.17.1-2-omv2014.0.i586, bridge-utils-1.5-3-omv2014.0.i586, grep-2.17-1-omv2014.0.i586 sono già installati

installazione di dracut-037-1-omv2014.0.i586.rpm
In preparazione...               #############################################
      1/1: dracut                #############################################
Executing: /usr/sbin/dracut -f /boot/initrd-3.13.8-nrjQL-desktop-1omv.img 3.13.8-nrjQL-desktop-1omv
00bootchart: Could not find command '/sbin/bootchartd'!
02caps: Could not find command 'capsh'!
05busybox: Could not find command 'busybox'!
dracut module 'network' will not be installed, because it's in the list to be omitted!
90multipath: Could not find command 'multipath'!
dracut module 'network' will not be installed, because it's in the list to be omitted!
dracut module 'cifs' depends on 'network', which can't be installed
95fcoe: Could not find command 'dcbtool'!
95fcoe: Could not find command 'fipvlan'!
95fcoe: Could not find command 'lldpad'!
95fcoe-uefi: Could not find command 'dcbtool'!
95fcoe-uefi: Could not find command 'fipvlan'!
95fcoe-uefi: Could not find command 'lldpad'!
95iscsi: Could not find command 'iscsistart'!
95iscsi: Could not find command 'iscsi-iname'!
95nbd: Could not find command 'nbd-client'!
dracut module 'nfs' depends on 'network', which can't be installed
97biosdevname: Could not find command 'biosdevname'!
dracut module 'aufs-mount' will not be installed, because it's in the list to be omitted!
*** Including module: bash ***
*** Including module: dash ***
*** Including module: xorgblacklist ***
*** Including module: modsign ***
dracut-install: ERROR: installing '/usr/bin/keyctl'
/usr/sbin/dracut-install -D /var/tmp/initramfs.O4TMjp /usr/bin/keyctl
*** Including module: i18n ***
*** Including module: ifcfg ***
*** Including module: drm ***
*** Including module: plymouth ***
*** Including module: btrfs ***
*** Including module: crypt ***
*** Including module: dm ***
Skipping udev rule: 64-device-mapper.rules
Skipping udev rule: 60-persistent-storage-dm.rules
Skipping udev rule: 55-dm.rules
*** Including module: dmraid ***
*** Including module: kernel-modules ***
*** Including module: lvm ***
grep: /var/tmp/initramfs.O4TMjp/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory
sed: can't read /var/tmp/initramfs.O4TMjp/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory
Skipping udev rule: 64-device-mapper.rules
Skipping udev rule: 56-lvm.rules
Skipping udev rule: 60-persistent-storage-lvm.rules
*** Including module: mdraid ***
Skipping udev rule: 63-md-raid-arrays.rules
Skipping udev rule: 64-md-raid-assembly.rules
*** Including module: resume ***
*** Including module: rootfs-block ***
*** Including module: terminfo ***
*** Including module: udev-rules ***
Skipping udev rule: 91-permissions.rules
Skipping udev rule: 80-drivers-modprobe.rules
*** Including module: systemd ***
*** Including module: usrmount ***
*** Including module: base ***
*** Including module: fs-lib ***
*** Including module: shutdown ***
*** Including modules done ***
*** Installing kernel module dependencies and firmware ***
*** Installing kernel module dependencies and firmware done ***
*** Resolving executable dependencies ***
*** Resolving executable dependencies done***
*** Stripping files ***
*** Stripping files done ***
*** Store current command line parameters ***
*** Creating image file ***
*** Creating image file done ***
      1/1: rimozione di dracut-036-5.i586
                                 #############################################

## Comment 5025
Date: 2014-04-09 10:06:58 +0000
From: @tpgxyz

(In reply to comment #25)
> i have tried with dracut-037-1
> 
> *** Including module: modsign ***
> dracut-install: ERROR: installing '/usr/bin/keyctl'

This is irrelevant to this issue, second why you have enabled modsign module, which is disabled by default ?

Anyways please attach output as a file of :
lsinitrd /boot/initrd-3.13.8-nrjQL-desktop-1omv.img > my_initrd_list

second boot in recovery mode (grub screen, choose adwanced and boot your lastest kernel with Recovery caption)

log in into VT and execute omv-bug-report.sh and attach it here.

These are needed for me to see if vconsole is set up, loadkeys are executed and whether initrd contain needed keymaps for you environment configuration.

## Comment 5026
Date: 2014-04-09 10:18:47 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to comment #26)
>.........second why you have enabled modsign module, which is disabled by default ?

i really don't know.
I have not touched anything deliberately

## Comment 5027
Date: 2014-04-09 10:19:27 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 405
initrd_list

> Attached file: my_initrd_list (application/octet-stream, 287844 bytes)
> Description:   initrd_list

## Comment 5028
Date: 2014-04-09 10:20:22 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 406
omv-bug-report.log

> Attached file: omv-bug-report.log (text/x-log, 1097728 bytes)
> Description:   omv-bug-report.log

## Comment 5029
Date: 2014-04-09 10:29:14 +0000
From: @tpgxyz

(In reply to comment #29)
> Created attachment 406 [details]
> omv-bug-report.log

Many thanks for this, now i have last (i hope so ;) question:

1. please rebuild your initrd with usr/sbin/dracut -f --hostonly /boot/initrd-3.13.8-nrjQL-desktop-1omv.img 3.13.8-nrjQL-desktop-1omv

2. Reboot and boot into recovery mode
3. log in into VT and run omv-bug-report.sh and attach output here
4. Check you VT keyboard settings (italian or not ;)

Looks like  /etc/vconsole.conf is not iniside your initrd, and i found that when running dracut in hostonly mode may help with proper VT keyboard settings.

## Comment 5030
Date: 2014-04-09 10:51:09 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

now i have itaian keyboard in VT !! :)

# /usr/sbin/dracut -f --hostonly /boot/initrd-3.13.8-nrjQL-desktop-1omv.img 3.13.8-nrjQL-desktop-1omv
Executing: /usr/sbin/dracut -f --hostonly /boot/initrd-3.13.8-nrjQL-desktop-1omv.img 3.13.8-nrjQL-desktop-1omv
00bootchart: Could not find command '/sbin/bootchartd'!
02caps: Could not find command 'capsh'!
05busybox: Could not find command 'busybox'!
dracut module 'network' will not be installed, because it's in the list to be omitted!
90multipath: Could not find command 'multipath'!
95fcoe: Could not find command 'dcbtool'!
95fcoe: Could not find command 'fipvlan'!
95fcoe: Could not find command 'lldpad'!
95fcoe-uefi: Could not find command 'dcbtool'!
95fcoe-uefi: Could not find command 'fipvlan'!
95fcoe-uefi: Could not find command 'lldpad'!
95iscsi: Could not find command 'iscsistart'!
95iscsi: Could not find command 'iscsi-iname'!
95nbd: Could not find command 'nbd-client'!
97biosdevname: Could not find command 'biosdevname'!
dracut module 'aufs-mount' will not be installed, because it's in the list to be omitted!
02caps: Could not find command 'capsh'!
05busybox: Could not find command 'busybox'!
90multipath: Could not find command 'multipath'!
95fcoe: Could not find command 'dcbtool'!
95fcoe: Could not find command 'fipvlan'!
95fcoe: Could not find command 'lldpad'!
95fcoe-uefi: Could not find command 'dcbtool'!
95fcoe-uefi: Could not find command 'fipvlan'!
95fcoe-uefi: Could not find command 'lldpad'!
95iscsi: Could not find command 'iscsistart'!
95iscsi: Could not find command 'iscsi-iname'!
95nbd: Could not find command 'nbd-client'!
*** Including module: bash ***
*** Including module: dash ***
*** Including module: xorgblacklist ***
*** Including module: i18n ***
*** Including module: ifcfg ***
*** Including module: drm ***
*** Including module: plymouth ***
*** Including module: kernel-modules ***
*** Including module: resume ***
*** Including module: rootfs-block ***
*** Including module: terminfo ***
*** Including module: udev-rules ***
Skipping udev rule: 91-permissions.rules
Skipping udev rule: 80-drivers-modprobe.rules
*** Including module: systemd ***
*** Including module: usrmount ***
*** Including module: base ***
*** Including module: fs-lib ***
*** Including module: shutdown ***
*** Including modules done ***
*** Installing kernel module dependencies and firmware ***
*** Installing kernel module dependencies and firmware done ***
*** Resolving executable dependencies ***
*** Resolving executable dependencies done***
*** Stripping files ***
*** Stripping files done ***
*** Store current command line parameters ***
*** Creating image file ***
*** Creating image file done ***

# cd /boot
# ls -alt
totale 136200
-rw-------  1 root root  9592000 apr  9 12:40 initrd-3.13.8-nrjQL-desktop-1omv.img

## Comment 5031
Date: 2014-04-09 10:52:14 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 407
omv-bug-report.log after --hostonly add

> Attached file: omv-bug-report.log (text/x-log, 1097260 bytes)
> Description:   omv-bug-report.log after --hostonly add

## Comment 5032
Date: 2014-04-09 12:25:44 +0000
From: @tpgxyz

(In reply to comment #31)
> now i have itaian keyboard in VT !! :)
> 

So as i suspected, rebuilding initrd in hostonly mode workarounds this bug.

## Comment 5033
Date: 2014-04-09 13:07:32 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i don't know if depends from --hostonly, but also modsign is not included in this way.

## Comment 5104
Date: 2014-04-14 04:29:56 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 414
bug report rc1 2014 build 3275

RC 1 build 3275 no change, the  issue still occurs

> Attached file: omv-bug-report.log.xz (application/x-xz, 57128 bytes)
> Description:   bug report rc1 2014 build 3275

## Comment 5182
Date: 2014-04-18 19:15:36 +0000
From: @tpgxyz

(In reply to comment #35)
> Created attachment 414 [details]
> bug report rc1 2014 build 3275
> 
> RC 1 build 3275 no change, the  issue still occurs

THis has been workarounded in dracut-037-4, which has been released after RC1 final iso build.

This bug is well explained here:
https://issues.openmandriva.org/show_bug.cgi?id=669

## Comment 5235
Date: 2014-04-20 14:12:38 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to comment #36)
> 
> THis has been workarounded in dracut-037-4, which has been released after
> RC1 final iso build.
> 

seems working here.
now vconsole keyboard is italian.

but, a little issue comes back:

backspace on login and password fields return "^H", while shift+backspace works

## Comment 5251
Date: 2014-04-21 21:28:34 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Sorry, i had a

systemctl restart systemd-vconsole-setup.service

in rc.local to workaround the issue.

removing it i still have default keyboard  :(

## Comment 5268
Date: 2014-04-22 07:15:39 +0000
From: @tpgxyz

(In reply to comment #38)
> Sorry, i had a
> 
> systemctl restart systemd-vconsole-setup.service
> 
> in rc.local to workaround the issue.
> 
> removing it i still have default keyboard  :(

Please run dracut -f

A real solution is describet on bug #669, so if rebuilding initrd with dracut-037-4 won't work then please add this to your /etc/default/grub cmdline

vconsole.font=latarcyrheb-sun16 vconsole.keymap=<yourkeymap> locale.LANG=<yourLANG>

## Comment 5277
Date: 2014-04-22 08:00:28 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to comment #39)
> 
> Please run dracut -f

$ rpm -aq | grep dracut
dracut-037-4-omv2014.0.x86_64

after dracut -f i had no luck.

> 
> A real solution is describet on bug #669, so if rebuilding initrd with
> dracut-037-4 won't work then please add this to your /etc/default/grub
> cmdline
> 
> vconsole.font=latarcyrheb-sun16 vconsole.keymap=<yourkeymap>
> locale.LANG=<yourLANG>

after i added in /etc/default/grub cmdline

vconsole.font=latarcyrheb-sun16 vconsole.keymap=it-latin1 locale.LANG=it_IT.UTF-8

it works

## Comment 5295
Date: 2014-04-22 10:39:37 +0000
From: @tpgxyz

(In reply to comment #40)

> after i added in /etc/default/grub cmdline
> 
> vconsole.font=latarcyrheb-sun16 vconsole.keymap=it-latin1
> locale.LANG=it_IT.UTF-8
> 
> it works

great to hear that, bad is that drakx-kbd-mouse-x11 needs development which is described on below bug report 
https://issues.openmandriva.org/show_bug.cgi?id=669

## Comment 5385
Date: 2014-04-23 15:36:28 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

I don't know why, but I have French keyboard (maybe command ask by TPG in bug #684) or I missed something :)

## Comment 5390
Date: 2014-04-23 16:38:09 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Joel,
i also am very confused at this point.
It also works for me.

I removed from the cmdline "vconsole.font = LatArCyrHeb-sun16 vconsole.keymap = it-latin1 locale.LANG = it_IT.UTF-8"

$ cat /proc/cmdline
BOOT_IMAGE = / boot/vmlinuz-3.13.10-nrjQL-desktop-1omv root = UUID = 259515f3-1BA1-46cb-841c-ro 5988cbb33193 logo.nologo acpi_osi = acpi_backlight Linux vendor = plymouth.enable = 0 resume = / dev/sda5

vconsole-setup.service seems ok.

# Systemctl status systemd-vconsole-setup.service
systemd-vconsole-setup.service - Setup Virtual Console
    Loaded: loaded (/ lib / systemd / system / systemd-vconsole-setup.service; static)
    Active: active (exited) since Wed 23/04/2014 18:23:56 am EDT; 7min August
      Docs: man: systemd-vconsole-setup.service (8)
            man: vconsole.conf (5)
  Main PID: 287 (code = exited, status = 0/SUCCESS)

April 23 18:23:56 localhost systemd [1]: Started Setup Virtual Console.

and keyboard is italian.

## Comment 5393
Date: 2014-04-23 16:57:51 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #43)
> Joel,
> i also am very confused at this point.
> It also works for me.
> 
> I removed from the cmdline "vconsole.font = LatArCyrHeb-sun16
> vconsole.keymap = it-latin1 locale.LANG = it_IT.UTF-8"
> 
> $ cat /proc/cmdline
> BOOT_IMAGE = / boot/vmlinuz-3.13.10-nrjQL-desktop-1omv root = UUID =
> 259515f3-1BA1-46cb-841c-ro 5988cbb33193 logo.nologo acpi_osi =
> acpi_backlight Linux vendor = plymouth.enable = 0 resume = / dev/sda5
> 
> vconsole-setup.service seems ok.
> 
> # Systemctl status systemd-vconsole-setup.service
> systemd-vconsole-setup.service - Setup Virtual Console
>     Loaded: loaded (/ lib / systemd / system /
> systemd-vconsole-setup.service; static)
>     Active: active (exited) since Wed 23/04/2014 18:23:56 am EDT; 7min August
>       Docs: man: systemd-vconsole-setup.service (8)
>             man: vconsole.conf (5)
>   Main PID: 287 (code = exited, status = 0/SUCCESS)
> 
> April 23 18:23:56 localhost systemd [1]: Started Setup Virtual Console.
> 
> and keyboard is italian.

hi Luca
same thing with you commands

strange
let us wait the boss :)

## Comment 5431
Date: 2014-04-24 11:58:32 +0000
From: @cris-b

*** Bug #689 has been marked as a duplicate of this bug. ***

## Comment 5464
Date: 2014-04-24 19:29:15 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

fresh install RC with all update

tty have english keyboard

rpm -aq | grep dracut
dracut-037-4-omv2014.0.x86_64

## Comment 5465
Date: 2014-04-24 19:30:03 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 448
report after all updates

> Attached file: omv-bug-report.log.xz (application/x-xz, 59280 bytes)
> Description:   report after all updates

## Comment 5466
Date: 2014-04-24 19:34:02 +0000
From: @cris-b

i think if you're doing it from install of rc1 you'll need to manually run localedrake and keyboardrake for the new setting to be added.

## Comment 5568
Date: 2014-04-28 05:29:03 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

build 3464 FC

virtual consoles are French keyboard

fixed for me

## Comment 5572
Date: 2014-04-28 06:32:50 +0000
From: @tpgxyz

Not closing this bug, just reducint severity to normal. Let's see if there are still some problems with VT and lang.

## Comment 5578
Date: 2014-04-28 06:48:08 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

build 3464
i can confirm italian keyboard in vconsole

## Comment 5585
Date: 2014-04-28 07:25:16 +0000
From: @itchka

Fixed

## Comment 5586
Date: 2014-04-28 07:27:30 +0000
From: @itchka

Reopening as per TPG
